frontend/src/components/App/App.css

```diff
@@ -50,68 +17,42 @@ body > iframe {
   }
 }
 
-/* CUSTOM APP STYLES */
-
-.App h2 {
-  padding: 0;
-}
-
-/* GENERAL PAGE STYLES */
-
-body > iframe {
-  display: none;
-}
-
-body {
-  margin: 0;
-  padding: 0;
-  overflow: hidden; /* Hide scrollbars */
-}
-
-/* NAVBAR - also in navbar.css */
-/* nav {
-  background-color: #3b3939;
-  padding: 0;
-  margin: 0 auto;
-  height: 70px;
-  z-index: 100;
+@media (prefers-reduced-motion: no-preference) {
+  #tap-in {
```

Comment:
Very cool, I wonder if we could add in 'tap to begin your journey' text to make it super clear for the user, not sure how to do that and not have it look too messy though